index.html

```diff
@@ -260,28 +260,27 @@ <h3>Partnerships</h3>
                         <div>
                             <div class="floe-front-news">
                                 <h3>News</h3>
+                                <h4>New Content Added to Inclusive Design Resources</h4>
+                                <time class="floe-date" datetime="2018-04-18">April 18, 2018</time>
+                                <p>
+                                    New content has been added to the <a href="https://guide.inclusivedesign.ca/">Inclusive Design Guide</a> as well as the <a href="https://handbook.floeproject.org/InclusiveLearning.html">Inclusive Learning Design Handbook</a>. We are excited to share this content with you as we continue to develop our thinking about inclusive design.
+                                </p>
+                                <p><a href="./news/2018-04-18-New-Content-Added-to-Inclusive-Design-Resources">Read more: New Content Added to Inclusive Design Resources</a></p>
```

Comment:
I think this url is missing the .html extension.

Reply:
I actually meant './news/2018-04-18-New-Content-Added-to-Inclusive-Design-Resources' is missing the .html I think the other link is better without the 'index.html'

Reply:
@michelled for consistency then, should I remove the index.html from the end of all the links to Guide and ILDH homepage?